### ISSUE TITLE
New-HPOVServerProfileTemplate small bug

### ISSUE THREAD

**Issue by scriptengine** [HewlettPackard/POSH-HPOneView#211]
### Expected Behavior
It would appear there is a period missing in the Send-HPOVRequest that is called by
New-HPOVServerProfileTemplate 
the error appears to be: -Hostname $_connection.Name

which I guess should be:
-Hostname $_**.**connection.Name

### Actual Behavior
```PowerShell
Send-HPOVRequest : Cannot validate argument on parameter 'Hostname'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
At C:\Program Files\WindowsPowerShell\Modules\HPOneView.300\HPOneView.300.psm1:63004 char:310
+ ... f (Send-HPOVRequest $c.networkUri -Hostname $_connection.Name).name,  ...
+                                                 ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Send-HPOVRequest], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Send-HPOVRequest
```
### Steps to reproduce
New-HPOVServerProfileTemplate @params -Connections $arrConnectionObjects 

### Version Information
PowerShell Library Version : 3.0.1191.2858
Library Path               : C:\Program Files\WindowsPowerShell\Modules\HPOneView.300

**Comment by ChrisLynchHPE** [comment]
Yes, this is a small bug.  No, the value you at suggesting isn't correct.  The Cmdlet was coded to support Multiple Appliance Connections within the Process code block, which really shouldn't have been.  `$_connection` should have been changed to `$ApplianceConnection`.

**Comment by scriptengine** [comment]
Is there a release planned for soon, that will include this fix?

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 3.0.1210.3013](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1210.3013).

Closing.
